utils/scripts/load-binary.sh

```diff
@@ -242,10 +242,9 @@ elif [ "$TARGET" = "waf_rule_set_v2" ]; then
 
 elif [ "$TARGET" = "waf_rule_set" ]; then
     assert_version_is_dev
-    curl --silent \
+    curl --fail --output "waf_rule_set.json" \
```

Comment:
I'd recommend hunting for _all_ uses of `curl` without `--fail` because they're ticking time bombs (but perhaps separately 😅)